pkg/k8shandler/kibana_handler/certificates.go

```diff
@@ -0,0 +1,140 @@
+package kibana_handler
```

Comment:
remove this file. this should only exist in CLO

Reply:
fixed